packages/create-block-tutorial-template/package.json

```
@@ -1,15 +1,16 @@
 {
 	"name": "@wordpress/create-block-tutorial-template",
-	"version": "2.33.0",
-	"description": "Template for @wordpress/create-block used in the official WordPress tutorial.",
+	"version": "3.0.0",
```

Version numbers for packages are automatically maintained so there should be no change here. Instead, this change should be marked with the `### Breaking Change` section in the CHANGELOG.md file. See more in: https://github.com/WordPress/gutenberg/tree/trunk/packages#maintaining-changelogs.

```suggestion
	"version": "2.33.0",
```

---

Ahh, that's why the linter was yelling at me 😅 All fixed.